### DISCUSSION TITLE
ESXI 8.0 U3 安装ImmortaWRT之后无法打开网页端

### DISCUSSION THREAD

**Discussion by cuitzeno** [immortalwrt/immortalwrt#1374] [Q&A]
### Describe the bug

下载https://downloads.openwrt.org/releases/23.05.3/targets/x86/64/openwrt-23.05.3-x86-64-generic-ext4-combined-efi.img.gz，通过StarWind V2V Converter创建EFI虚拟机，开机之后通过vi /etc/config/network修改为同网段IP地址之后重新启动，用浏览器无法打开

### ImmortalWrt version

23.05.3

### ImmortalWrt release

na

### ImmortalWrt target/subtarget

na

### Device

j4125 X86

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by toss-a** [comment]
创建ESXi 7.0 U2规格虚拟机 使用BIOS引导即可解决
